charts/substra-backend/templates/deployment-server.yaml

```diff
@@ -140,19 +140,17 @@ spec:
           value: {{ .Values.postgresql.postgresqlPassword }}
         - name: DATABASE_HOST
           value: {{ .Release.Name }}-postgresql
-      {{- if eq .Values.backend.settings "prod" }}
       - name: init-collectstatic
         image: "{{ .Values.backend.image.repository }}:{{ .Values.backend.image.tag }}"
         command: ['python', 'manage.py', 'collectstatic', '--noinput']
         env:
           - name: DJANGO_SETTINGS_MODULE
-            value: backend.settings.{{ .Values.backend.settings }}
+            value: backend.settings.prod
```

Comment:
Why is this now a fixed value ?
And is it necessary to keep it because we pass this as `--env DJANGO_SETTINGS_MODULE=backend.settings.{{ .Values.backend.settings }}` at these lines https://github.com/SubstraFoundation/substra-backend/blob/e6a0cb62abcb3dab2d5839fcf0094d55a700dabb/charts/substra-backend/templates/deployment-server.yaml#L37 and https://github.com/SubstraFoundation/substra-backend/blob/e6a0cb62abcb3dab2d5839fcf0094d55a700dabb/charts/substra-backend/templates/deployment-server.yaml#L39

Reply:
This is not a fix value. As said previously in the Aurelien's comment, we need to use `prod` settings to generate static files, as we expose now the server with uswgi in both cases

Reply:
@AlexandrePicosson Good for you ?

Reply:
Yes, I was just not sure of what this env variable would change and thought it might activate some security settings we do not want in dev.